packages/recommendations-core/src/__tests__/getRecommendations.test.ts

```diff
@@ -57,12 +55,12 @@ function createRecommendationsClient() {
   };
 }
 
-describe('useRecommendations', () => {
+describe('getRecommendations', () => {
   test('calls the correct index for "related-products"', async () => {
     const { index, searchClient } = createRecommendationsClient();
     const props = {
       model: 'related-products' as RecommendationModel,
-      searchClient,
+      searchClient: searchClient as any,
```

Comment:
That's just temporary because the typings for the yet non-existant Recommendations API client don't exist. Since the expected search client and its typings are going to change very soon, I didn't bother hiding the typing issue.